packages/openneuro-components/src/header/LandingExpandedHeader.tsx

```diff
@@ -1,12 +1,14 @@
-import React, { useEffect, useRef } from 'react'
+import React from 'react'
 
 import { Button } from '../button/Button'
 import { ModalityCube } from '../modality-cube/ModalityCube'
+import { Input } from '../input/Input'
 import { cubeData } from '../mock-content/modality-cube-content.jsx'
 import orcidIcon from '../assets/orcid_24x24.png'
 import { AggregateCount } from '../aggregate-count/AggregateCount'
 
 import { frontPage } from '../mock-content/front-page-content.jsx'
+import { FrontFacetExample } from '../facets/Facet.stories'
 
```

Comment:
The cubeWrap mostly looks great, but gets distorted at some view widths.

Reply:
Maybe a container of set width? i'm not sure how to fix this

Reply:
We have to do some additional responsive work. I think at smaller viewports we will stack them and get rid of the cube. undecided at the moment.